dd-sdk-android/src/main/kotlin/com/datadog/android/error/internal/DatadogExceptionHandler.kt

```diff
@@ -63,6 +65,21 @@ internal class DatadogExceptionHandler(
     // region Internal
 
     private fun createLog(thread: Thread, throwable: Throwable): Log {
```

Comment:
it seems like we only need `thread.name`, maybe we can pass the name instead of `Thread`?
in order to avoid retaining `Thread` instance and for easiness in tests